FreeRTOS_IP.c

```diff
@@ -2625,10 +2624,6 @@ static eFrameProcessingResult_t prvProcessIPPacket( IPPacket_t * pxIPPacket,
              * protocol (Layer 3 or 4) header will be aligned, which is the convenience
```

Comment:
This comment should be removed too, I think

Reply:
Yes sure, the entire comment can be removed now. The same comment can be found earlier in this source file.

Reply:
Removed the comment